typescript/ec2-ssm-local-zone/example-localzone-stack.ts

```diff
@@ -0,0 +1,204 @@
+import { Stack, StackProps, CfnOutput } from 'aws-cdk-lib';
+import {
+  aws_iam as iam,
+  aws_ec2 as ec2
+} from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+// TODO: Change these to suitable values
+const PARENT_REGION_AZ = 'eu-north-1a';
+const LOCAL_ZONE_AZ = 'eu-north-1-hel-1a';
+
+const VPC_CIDR = '172.31.100.0/24';
+const SUBNET_SIZE = 26;
+
+// Amazon Linux 2022 https://docs.aws.amazon.com/linux/al2022/ug/what-is-amazon-linux.html
+const al2022Ami = ec2.MachineImage.fromSsmParameter('/aws/service/ami-amazon-linux-latest/al2022-ami-kernel-5.15-x86_64');
```

Comment:
Any reason not using this instead?

```ts
const al2022Ami = new ec2.AmazonLinuxImage({
  generation: ec2.AmazonLinuxGeneration.AMAZON_LINUX_2022,
})
```

Reply:
OK there seems to be a bug in `AMAZON_LINUX_2022` that always returns kernel 5.10 whereas only 5.15 is currently available. 

https://github.com/aws/aws-cdk/blob/cea1039e3664fdfa89c6f00cdaeb1a0185a12678/packages/%40aws-cdk/aws-ec2/lib/machine-image.ts#L392

https://github.com/aws/aws-cdk/blob/cea1039e3664fdfa89c6f00cdaeb1a0185a12678/packages/%40aws-cdk/aws-ec2/lib/machine-image.ts#L467

I'll see what we can do as a workaround before we fix it from the upstream.

Reply:
Updated!